test-engineering/contract-tests/partner/README.md

```diff
@@ -31,18 +31,84 @@ You can run the service using `docker compose` from the root directory:
 docker compose run -p 5000:5000 partner
 ```
 
-## Tiles API
+## API
 
-Example request:
+Once the API service is running, API documentation can be found at http://0.0.0.0:5000/docs
+
+### Records
+
+Example GET request:
+
+```text
+curl \
+  -X 'GET' \
+  -H 'accept: application/json' \
+  'http://0.0.0.0:5000/records/'
+```
+
+Example GET response body:
+
+```json
+{
+  "records": [
+    {
+      "method": "GET",
+      "headers": [
+        {
+          "name": "host",
+          "value": "0.0.0.0:5000"
+        },
+        {
+          "name": "user-agent",
+          "value": "curl/7.79.1"
+        },
+        {
+          "name": "accept",
+          "value": "application/json"
+        }
+      ],
+      "path": {
+        "endpoint": "desktop"
```

Comment:
[1434e96](https://github.com/mozilla-services/contile/pull/415/commits/1434e961b2d17be67f809b0c34b73eb0448899ed) - Apologies, I neglected to update the response body in this document. The endpoint does return '/tilesp/desktop'. I also fixed the next line, since 'query' is now 'query_parameters'.